client/src/app/videos/+video-watch/video-watch.component.scss

```diff
@@ -64,6 +64,34 @@ $other-videos-width: 260px;
   }
 }
 
+@keyframes fade {
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+.video-element-wrapper-popup {
+  position: fixed;
+  bottom: 20px;
+  right: 20px;
+  z-index: 30;
+  animation: fade 1.5s;
+  box-shadow: 0 5px 10px 0 rgba(0,0,0,.3),0 1px 8px -3px rgba(0,0,0,.26),0 2px 16px 0 rgba(0,0,0,.28);
+
+  /deep/ .video-js {
+    width: 280px !important;
+    height: 160px !important;
+  }
+
+  /deep/ .vjs-peertube-displayed,
+  /deep/ .vjs-theater-control {
+    display: none !important;
```

Comment:
theater icon should already be hidden https://github.com/Chocobozzz/PeerTube/blob/develop/client/src/sass/player/peertube-skin.scss#L471

Reply:
it is not.
Why should it be? the option to go into theater mode is available, but in the popup player, it doesn't make sense and only takes space